.github/workflows/android_builds.yml

```diff
@@ -6,6 +6,7 @@ env:
   GODOT_BASE_BRANCH: master
   SCONSFLAGS: platform=android verbose=yes warnings=extra werror=yes --jobs=2 module_text_server_fb_enabled=yes
   SCONS_CACHE_LIMIT: 4096
+  ANDROID_NDK_VERSION: 21.1.6352462
```

Comment:
Had to pin this version for now as Gradle would still try to use `21.1.6352462` even if I pre-installed `21.3.6528147` and set it as `ANDROID_NDK_ROOT`. Hopefully #44949 should solve that.